simulation/mdp.py

```diff
@@ -0,0 +1,124 @@
+"""
+Helper functions for abstractin a simulation as an MDP (Markov Decision Process)
+including converting between simulation data and MDP constructs and recording of
+states/observations, actions, and rewards.
+"""
+
+import torch
+import jsbsim_properties as prp
+
+"""
+Extracts agent state data from the sim.
+"""
+def state_from_sim(sim):
+  state = torch.zeros(15,)
+
+  M_TO_FT = 0.3048
+  
+  # position
+  state[0] = sim[prp.lng_travel_m] * M_TO_FT # x
+  state[1] = sim[prp.lat_travel_m] * M_TO_FT # y
```

Comment:
Perhaps consider not including x,y position in the state if the waypoint position is relative to current position